realm/realm-library/src/objectServer/java/io/realm/RealmAppConfiguration.java

```diff
@@ -388,6 +413,14 @@ public Builder syncRootDirectory(File rootDir) {
             return this;
         }
 
+        // FIXME Naming: Does it clash with Realm sync concepts and set up wrong expectations?
```

Comment:
We don't have a similar name. I find this name really exotic and not very readable (as a non-native english speaker), but given that I don't really have a good alternative we might as well adopt this I guess. It does has the benefit that it will be familiar to anyone coming to it from the Mongo side.

Reply:
Ok, will keep it then.